exercises/practice/bottle-song/.meta/design.md

```diff
@@ -0,0 +1,11 @@
+# Design Decisions
+
+## Wording
+
+Since the "verse" term used in the canonical specifications is quite ambiguous, it has been replaced by:
```

Comment:
I don't see the ambiguity in the specs, they refer to "verse" as what you refer to as "stanza". 
Is it because the expected values are a list of strings ("lines")? Usually, those are meant to correspond to multiline strings, which JSON represents poorly.

Maybe I'm missing something, but at the moment I would recommend going back to the "verse" term

Reply:
I see where you're coming from now.
I think in this case we are talking about a song rather than a poem, so "verse" would likely be translated to "couplet". 

In any case, both the exercise description and tests "verse" talk about the same thing, so there is no ambiguity there, in fact, introducing "stanza" in tests would _create_ ambiguity. Let's stick to "verse" :)